src/VirtoCommerce.CartModule.Core/ModuleConstants.cs

```diff
@@ -14,6 +17,46 @@ public static class Permissions
 
                 public static string[] AllPermissions = new[] { Read, Create, Access, Update, Delete };
             }
-        }
+        }
+
+        public static class Settings
+        {
+            public static class General
+            {
+                public static readonly SettingDescriptor EnableDeleteObsoleteCarts = new SettingDescriptor
+                {
+                    Name = "Cart.EnableDeleteObsoleteCarts",
+                    GroupName = "Cart|General",
+                    ValueType = SettingValueType.Boolean,
+                    DefaultValue = true
+                };
+
+                public static readonly SettingDescriptor PortionDeleteObsoleteCarts = new SettingDescriptor
+                {
+                    Name = "Cart.PortionDeleteObsoleteCarts",
+                    GroupName = "Cart|General",
+                    ValueType = SettingValueType.Integer,

```

Comment:
PositiveInteger here.

Reply:
I thought about it, but need to raise dependency more